recipes/sdl_net/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class SdlnetConan(ConanFile):
+    name = "sdl_net"
+    description = "A networking library for SDL"
+    license = "Zlib"
+    topics = ("sdl2", "sdl2_net", "sdl", "sdl_net", "net", "networking")
+    homepage = "https://www.libsdl.org/projects/SDL_net"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake", "cmake_find_package"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def requirements(self):
+        self.requires("sdl/2.0.16")
+
+    def validate(self):
+        if self.settings.compiler == "Visual Studio" and self.options.shared:
+            raise ConanInvalidConfiguration("sdl_net is not supported with Visual Studio")
+        # TODO: check that major version of sdl_net is the same than sdl (not possible yet in validate())
```

Comment:
You can do this check in `build` for now (and keep this comment as a FIXME)

Reply:
Done: https://github.com/conan-io/conan-center-index/pull/7355/commits/df654215b9821435e55fe9e47e7bb253f937d356